lib/properties/individu-properties.ts

```diff
@@ -268,11 +268,20 @@ export default {
   }),
 
   date_debut_chomage: new DateProperty({
-    question: ({ individu }) => {
-      return `Quand ${IndividuMethods.label(
-        individu,
-        "avoir"
-      )} commencé à être au chômage ?`
+    question: ({ individu, abtestingService }) => {
+      const abtestingEnvironment = abtestingService.getEnvironment()
+      const version = abtestingEnvironment.question_debut_chomage
```

Comment:
Pour que ce soit fonctionnel, il faut passer par la propriété `value` ici :
 ```suggestion
       const version = abtestingEnvironment.question_debut_chomage.value
```